Makefile

```diff
@@ -293,3 +294,30 @@ operator-registry-image:
 
 	$(IMAGE_BUILD_CMD) --build-arg MANIFEST_LOCATION=$(MANIFESTS_DIR) -t $(LOCAL_OPERATOR_REGISTRY_IMAGE) -f images/operator-registry/Dockerfile.registry .
 	$(DOCKER_RUNTIME) push $(LOCAL_OPERATOR_REGISTRY_IMAGE)
+
+# TODO(jkyros): I can't migrate the whole operator to the new operator-sdk, there's a bunch of stuff we have to re-arrange, the project layout is different, 
```

Comment:
Consider creating a Jira card to track this effort in the backlog and link the card here in TODO.